.circleci/ci/accessibility-axe.js

```diff
@@ -0,0 +1,70 @@
+/*******************************************************************************
+ * Copyright 2023 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+const fs = require('fs');
+const AxeBuilder = require('@axe-core/webdriverjs');
+const WebDriver = require('selenium-webdriver');
+const chrome = require('selenium-webdriver/chrome');
+const { createHtmlReport } = require('axe-html-reporter');
+
+
+const calculateAccessibility = async () => {
+
+    const options = new chrome.Options();
+    const driver = new WebDriver.Builder().forBrowser('chrome').setChromeOptions(options).build();
+
+    try {
+        await driver.get('http://localhost:4502/content/dam/formsanddocuments/core-components-it/samples/wizard/repeatability/jcr:content?wcmmode=disabled');
+
+        await new Promise((resolve) => setTimeout(resolve, 4000));
+//        await driver.findElement(WebDriver.By.xpath("//button[@aria-expanded='false']/coral-accordion-item-label[contains(text(),'Sign in locally')]")).click();
+        const usernameInput = await driver.findElement(WebDriver.By.id('username'));
+        await usernameInput.sendKeys('admin');
```

Comment:
Please read the username/password from environment variables

Reply:
done